sdcm/stress_thread.py

```diff
@@ -158,11 +158,23 @@ def _run_stress(self, node, loader_idx, cpu_idx, keyspace_idx):
                                      cs_log_filename=log_file_name,
                                      loader_idx=loader_idx, cpu_idx=cpu_idx), \
                 CassandraStressEventsPublisher(node=node, cs_log_filename=log_file_name):
-
-            result = node.remoter.run(cmd=node_cmd,
-                                      timeout=self.timeout,
-                                      ignore_status=True,
-                                      log_file=log_file_name)
+            result = None
+            try:
+                result = node.remoter.run(cmd=node_cmd,
+                                          timeout=self.timeout,
+                                          log_file=log_file_name)
+            except Exception as exc:  # pylint: disable=broad-except
+                #  pylint: disable=no-member
+                if hasattr(exc, 'result') and exc.result.failed:
+                    output = exc.result.stderr
+                    if not output:
+                        output = exc.result.stdout
+                    errors_str = f'Stress command return error: {output}'
```

Comment:
this is bad! we store the output of the command in the log and it is huge! why do you want to output again and in the event? Anyway, I don't understand why are you complicating this? this case can be finished by:

```
CassandraStressEvent(type='finish', node=str(node), stress_cmd=stress_cmd,
                                     log_file_name=log_file_name, severity=Severity.CRITICAL,
                                     errors=errors_str, severity=Severity.CRITICAL if result.exit_status else  Severity.NORMAL)
```
(the type is not so important, currently, we don't use it)